Dịch "bert_vn" - Phần 5 (#2633)
aivivn/d2l-vn: chapter_natural-language-processing-pretraining/bert_vn.md

(e.g., "great" is selected to be masked and predicted in "this movie is great"), in the input it will be replaced with:
-->

Trong tác vụ tiền huấn luyện này, 15% các token sẽ được lựa chọn ngẫu nhiên để làm các token có mặt nạ cho việc dự đoán.
Để dự đoán một token có mặt nạ mà không ăn gian bởi việc sử dụng nhãn, một hướng tiếp cận thẳng đó là luôn luôn thay thế nó với một token đặc biệt “&lt;mask&gt;” trong chuỗi đầu vào BERT.
Tuy nhiên, token đặc biệt nhân tạo “&lt;mask&gt;” sẽ không bao giờ xuất hiện trong tinh chỉnh. 

> Để tránh xảy ra sự không tương đồng giữa tiền huấn luyện và tinh chỉnh, nếu một token được gắn mặt nạ dùng cho đự đoán (ví dụ, "great" được chọn để gắn mặt nạ và dự đoán trong "this movie is great"), trong đầu vào nó sẽ được thay thế bởi:

Reviewer: ```suggestion
Để tránh xảy ra sự không đồng nhất giữa tiền huấn luyện và tinh chỉnh, nếu một token được che mặt nạ để dự đoán (ví dụ, "great" được chọn để che mặt nạ và dự đoán trong "this movie is great"), trong đầu vào nó sẽ được thay thế bởi:
```

* the unchanged label token for 10% of the time (e.g., "this movie is great" becomes "this movie is great").
-->

> * một token “&lt;mask&gt;” đặc biệt cho 80% thời gian (ví dụ, "this movie is great" trở thành "this movie is &lt;mask&gt;”);
> * một token ngẫu nhiên cho 10% thời gian (ví dụ, "this movie is great" trở thành "this movie is drink");
> * một token nhãn không thay đổi cho 10% thời gian (ví dụ, "this movie is great" trở thành "this movie is great").

Reviewer: ```suggestion
* một token “&lt;mask&gt;” đặc biệt, xác suất 80% (ví dụ, "this movie is great" trở thành "this movie is &lt;mask&gt;”);
* một token ngẫu nhiên, xác suất 10% (ví dụ, "this movie is great" trở thành "this movie is drink");
* chính token đó, xác suất 10% (ví dụ, "this movie is great" trở thành "this movie is great").
```
thế này có chuẩn không nhỉ?

<!--
Note that for 10% of 15% time a random token is inserted.
This occasional noise encourages BERT to be less biased towards the masked token (especially when the label token remains unchanged) in its bidirectional context encoding.
-->

> Lưu ý rằng cho 10% của 15% thời gian một token ngẫu nhiên được chèn vào.

Reviewer: không hiểu 15% ở đâu ra @@.

> Nhiễu trừ bị này khuyến khích BERT bớt đi thiên kiến với token có mặt nạ (đặc biệt khi token nhãn duy trì trạng thái không đổi) trong giải mã ngữ cảnh hai chiều.

Reviewer: ```suggestion
Nhiễu không thường xuyên này giúp BERT bớt đi thiên kiến với token có mặt nạ (đặc biệt khi token nhãn không đổi) trong giải mã ngữ cảnh hai chiều.
```

The output is the prediction results at these positions.
-->

> Ta lập trình lớp `MaskML` sau để dự đoán token có mặt nạ trong tác vụ mô hình hóa ngôn ngữ có mặt nạ của tiền huấn tuyện BERT.

Reviewer: ```suggestion
Ta lập trình lớp `MaskML` sau để dự đoán token có mặt nạ trong tác vụ mô hình hóa ngôn ngữ có mặt nạ khi tiền huấn tuyện BERT.
```

> Việc dự đoán này sử dụng một MLP một-tầng-ẩn (`self.mlp`).

Reviewer: ```suggestion
Việc dự đoán này sử dụng một Perceptron một-tầng-ẩn (`self.mlp`).
```

> Trong suy luận truyền xuôi, nó nhận vào hai đầu vào: một kết quả được mã hóa của `BERTEncoder` và token vị trí cho việc dự đoán.

Reviewer: ```suggestion
Suy luận truyền xuôi nhận hai đầu vào: kết quả mã hóa của `BERTEncoder` và vị trí token để dự đoán.
```
